packages/poisoning/package.json

```diff
@@ -2,33 +2,33 @@
   "name": "@fast-check/poisoning",
   "description": "Set of utilities to ease detection and revert of poisoning",
   "version": "0.1.0",
-  "type": "commonjs",
+  "type": "module",
   "main": "lib/main.js",
```

Comment:
Should I keep it referencing the cjs version for backward compatibility (pre-node 12)? So far it does not seem to be a good option, but at the same time I don't know if this field still makes sense today...